colour/appearance/ciecam02.py

```diff
@@ -622,8 +626,7 @@ def chromatic_induction_factors(n: ArrayLike) -> NDArrayFloat:
 
     n = as_float_array(n)
 
-    with sdiv_mode():
```

Comment:
Yeah I was going to go through and put some of these back in where I moved them. there is one relatively important one I want to remove because the values that can cause an issue are generated in the same function context and can be made safer.

Reply:
Reverted